src/test/run-pass-fulldeps/qquote.rs

```diff
@@ -8,6 +8,7 @@
 // option. This file may not be copied, modified, or distributed
 // except according to those terms.
 
+// ignore-cross-compile
```

Comment:
I believe the `// ignore-test` line in that file means that the test is unconditionally skipped.  I verified the the test is skipped on one of the Linux buildbot builders.  In any case, it needs `// ignore-cross-compile` because it uses the `syntax` crate.